includes/options/views/class-amp-setup-wizard-submenu-page.php

```diff
@@ -0,0 +1,223 @@
+<?php
+/**
+ * AMP setup wizard page.
+ *
+ * @package AMP
+ * @since @todo NEW_ONBOARDING_RELEASE_VERSION
+ */
+
+/**
+ * AMP setup wizard submenu page class.
+ *
+ * @since @todo NEW_ONBOARDING_RELEASE_VERSION
+ */
+class AMP_Setup_Wizard_Submenu_Page {
```

Comment:
Classes should default to `final`, unless we have a specific use case we want to open up extensibility for.
```suggestion
final class AMP_Setup_Wizard_Submenu_Page {
```